jena-fuseki2/jena-fuseki-webapp/src/main/java/org/apache/jena/fuseki/mgt/Backup.java

```diff
@@ -84,6 +84,10 @@ public static void backup(Transactional transactional, DatasetGraph dsg, String
      * @see #backup(Transactional, DatasetGraph, String)
      */
     private static void backup(DatasetGraph dsg, String backupfile) {
+        if (dsg == null) {
```

Comment:
Stumbled on this while writing some new unit tests around both `/$/backup/<dataset>` and `/$/compact/<dataset>`

This function didn't verify that `dsg` was not `null` so when it tried to interact with the `ConcurrentHashMap` that stores active backups it would NPE because concurrent maps do not permit `null` keys